Dockerfile

```diff
@@ -39,6 +39,6 @@ RUN mkdir -p /var/log/imqs/
 RUN mkdir -p /var/imqs/secrets
 COPY --from=builder /build/imqsauth /opt/imqsauth
 EXPOSE 80
-ENTRYPOINT ["wait-for-nc.sh", "config:80", "--", "wait-for-postgres.sh", "db", "/opt/imqsauth"]
+ENTRYPOINT ["wait-for-nc.sh", "config:80", "--", "/opt/imqsauth"]

```

Comment:
Why did this need to be changed?

Reply:
This was some previous thing that we did related to preparing IMQS to run as a standalone stack with external postgres server (for DTPW). Vos' commit